src/dbus_python_client_gen/_invokers.py

```diff
@@ -340,7 +341,7 @@ def builder(namespace):
            types.new_class(
               "Methods",
               bases=(object,),
-              exec_body=method_builder(spec)
+              exec_body=method_builder(spec, timeout)
```

Comment:
Why doesn't ```prop_builder``` get timeout as well? Ultimately, it becomes a method call of some sort.